backend/app/api/endpoints/openapi_responses.py

```
@@ -145,7 +148,591 @@ def _task_to_response_object(
+    # For list input, get the last user message
+    for item in reversed(input_data):
+        if isinstance(item, InputItem) and item.role == "user":
+            return item.content
+        elif isinstance(item, dict) and item.get("role") == "user":
+            return item.get("content", "")
+
+    # If no user message found, return empty string
+    return ""
+
+
+def _check_team_supports_direct_chat(db: Session, team: Kind, user_id: int) -> bool:
```

Ruff (0.14.10)

Unused function argument: `user_id`

(ARG001)